ArchiveLibrary/keywords.py

```diff
@@ -93,14 +97,15 @@ def create_tar_from_files_in_directory(self, directory, filename, sub_directorie
         `sub_directories` Shall files in sub-directories be included - True by default.        
         """
         tar = tarfile.open(filename, "w")
-        files = return_files_lists(directory, sub_directories)
 
+        files = return_files_lists(directory, sub_directories)
         for filepath, name in files:
             tar.add(filepath, arcname=name)
 
         tar.close()
 
-    def create_zip_from_files_in_directory(self, directory, filename, sub_directories=False, compression="stored"):
+    @classmethod
```

Comment:
Why the classmethod?

Reply:
In this case create_zip_from_files_in_directory doesn't use `self` and it needs of `compression` dict.